Library/Homebrew/sandbox.rb

```diff
@@ -8,6 +8,14 @@ def self.available?
     OS.mac? && File.executable?(SANDBOX_EXEC)
   end
 
+  # there're times sandbox is inconvenience
```

Comment:
Maybe `should not be used`? As actually, we can use it, it's just inconvenience.